test/parallel/test-cluster-dgram-reuse.js

```diff
@@ -37,4 +37,4 @@ function close() {
 }
 
 for (let i = 0; i < 2; i++)
-  dgram.createSocket({ type: 'udp4', reuseAddr: true }).bind(common.PORT, next);
+  dgram.createSocket({ type: 'udp4', reuseAddr: true }).bind(0, next);
```

Comment:
I think this would be better if you remove the `for` loop and have two separate `bind()` calls. The outer call to `bind()` would use port 0 and the other call would happen in the callback for that `bind()`, using `.address().port` to get the port of the outer bind and reuse it.